gnome-initial-setup/pages/live-chooser/gis-live-chooser-page.c

```diff
@@ -51,6 +51,19 @@ typedef struct _GisLiveChooserPagePrivate GisLiveChooserPagePrivate;
 G_DEFINE_TYPE_WITH_PRIVATE (GisLiveChooserPage, gis_live_chooser_page, GIS_TYPE_PAGE);
 
 static void
+disable_chrome_auto_download (GisLiveChooserPage *page)
+{
+  GError *error = NULL;
+
+  if (!gis_pkexec (DATADIR "/eos-google-chrome-helper/eos-google-chrome-system-helper.py",
+                   NULL, &error))
+    {
+      g_warning ("Failed to disable Chrome auto-download: %s\n", error->message);
```

Comment:
I found many occurrences that are also upstream, how should we fix them? preparing patch for upstream or just patch our fork? @cosimoc